### PR TITLE
BAQE-902 - Added testcases for negative values. Whitespace remover added

### PR DESCRIPTION
**Pull request by Bonuseto**
Hi Jozef @jomarko , jenkins execute full downstream build

### CODE REVIEW

**Comment by Bonuseto** [comment]
jenkins execute full downstream build

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/constraint/common/typed/years/months/YearsMonthsValueConverter.java [hunk 1]
```diff
@@ -42,8 +42,9 @@ public YearsMonthsValueConverter(final ClientTranslationService translationServi
 
     public YearsMonthsValue fromDMNString(final String dmnString) {
 
+        String dmnStringWithoutWhiteSpaces = dmnString.replaceAll("\\s","");
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/constraint/common/typed/years/months/YearsMonthsValueConverter.java]: As discussed on todays meeting. Please remove this line.

**Comment by Bonuseto** [comment]
Hi Jozef @jomarko , incorrect line removed, tests ignored

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/types/listview/constraint/common/typed/years/months/YearsMonthsValueConverterTest.java [hunk 2]
```diff
@@ -58,41 +59,104 @@ public void testFromDMNStringYearsMonthsSingleDigit() {
...
         testFromDMNString(input, "", "12");
     }
 
+    @Test
+    public void testFromDMNStringYearsMonthsOnlyMonthNegative() {
+
+        final String input = "duration(\"P-12M\")";
+        testFromDMNString(input, "", "-12");
+    }
+ @Ignore
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/types/listview/constraint/common/typed/years/months/YearsMonthsValueConverterTest.java]: Please keep one empty line between test methods and proper indentation.

```java
@Ignore("DROOLS-4082")
@Test
public void testFromDMNStringWhiteSpacesStartAndEnd() {
    final String input = "duration( \"P1Y\" )";
    testFromDMNString(input, "1", "");
}
```

Please apply the same for `testFromDMNStringWhiteSpacesStart` and `testFromDMNStringWhiteSpacesEnd`.

**Comment by Bonuseto** [comment]
Hi @jomarko ,changes are done

**Comment by jomarko** [comment]
@Bonuseto sorry, I forgot, last thing, please remove the [DO NOT MERGE] from PR description.

**Comment by Bonuseto** [comment]
Hi @manstis could you merge this one please?

**Comment by Bonuseto** [comment]
@manstis Thank you!

**Pull request merged**